go/oasis-test-runner/scenario/e2e/stake_cli.go

```diff
@@ -81,6 +82,8 @@ var (
 	// we can expect the amount of reclaimed base units to equal the number of reclaimed escrow
 	// shares.
 	reclaimEscrowAmount int64 = reclaimEscrowShares
+
+	ctx = context.Background()
```

Comment:
Having a global like this sounds bad as `ctx` is commonly used and this makes it global in the whole `e2e` package.

Reply:
Agreed. I've made it per test case now.